code/modules/jobs/job_types/peasants/miner.dm

```diff
@@ -43,7 +43,8 @@
 	H.adjust_skillrank(/datum/skill/misc/athletics, 3, TRUE)
 	H.adjust_skillrank(/datum/skill/craft/traps, 1, TRUE)
 	H.adjust_skillrank(/datum/skill/craft/engineering, 2, TRUE)
-	H.adjust_skillrank(/datum/skill/craft/smelting, 4, TRUE)
+	H.adjust_skillrank(/datum/skill/craft/blacksmithing, 1, TRUE)//I assume they would know a thing or two about smithing
+	H.adjust_skillrank(/datum/skill/craft/smelting, 4, TRUE)//if they are such good smelters
```

Comment:
i would suggest a random pick 2,3 smelting, their job is MINING nor smithing or blacksmithing, but they can be average or skilled depending if they focused or not on it

For those who say otherwise, there is no way a miner would be better at smelting than the armorer and weaponsmith that smelt all the time and better than the artificer who literally smelts bronze all the time as well.